OWNERS

```diff
@@ -10,3 +10,4 @@ kemppeterson@google.com
 shobs@google.com
 swast@google.com
 tbergeron@google.com
+mlaurencechen@google.com
```

Comment:
this file is so outdated. you can remove ashleyxu, bmil, henryjsolberg, hormati and kemppeterson.

Reply:
done!